Common/Data/Custom/Tiingo/TiingoDailyData.cs

```diff
@@ -72,31 +73,31 @@ public override DateTime EndTime
         /// The actual (not adjusted) open price of the asset on the specific date
         /// </summary>
         [JsonProperty("open")]
-        public decimal Open { get; set; }
+        public new decimal Open { get; set; }
```

Comment:
yea, I guess I didn't think about how we'll add the `JsonProperty` attributes to the base class properties -- I think these `get/set` implementations **must** patch through to the `base`:

``` csharp
[JsonProperty("open")]
public new decimal Open
{
    get { return base.Open; }
    set { base.Open = value; }
}
```

otherwise we'll have separate storage, one for this who see this as `TradeBar` and another for those who see this as `TiingoDailyData`

Reply:
Another option is to make open/high/low/close/volume/period all overrideable -- then you can decorate them directly without the layer of indirection. This also would enable this same behavior for any users attempting to subclass -- thoughts?